recipes/responder/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "responder" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c2232012ce346104a4f0e225731cab24bf80ba460b8636d1a6c05343d1c95b52
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
I once heard (but I might be wrong) that it could be better to use version constraints in the requirements below (`- python >=3.6`), rather than using the `skip` directive.

Reply:
IIUC (for arch specific builds) you should just use plain `python` and constrain with selectors as above.

I'm not sure I follow entirely but details are in https://github.com/conda-forge/conda-smithy/issues/904

> From @isuruf on the conda-forge gitter channel: gitter.im/conda-forge/conda-forge.github.io?at=5bbb8adb271506518dd861b7
>
>> python matrix builds are used if you have just - python as a requirement.
>> python >= 3.5 means conda-build can use any python version >= 3.5 as a run requirement, but what we actually want is the python version that it built against as the run requirement”

Reply:
oh yes, you are correct, thanks for pointing me to those discussions